src/Dfe.PlanTech.Web/ViewComponents/CategorySectionViewComponent.cs

```diff
@@ -68,22 +79,37 @@ private static void SetCategorySectionDtoTagWithCurrentStatus(Category category,
         }
         else
         {
-            categorySectionDto.TagColour = TagColour.Grey.ToString();
+            categorySectionDto.TagColour = TagColour.Grey;
             categorySectionDto.TagText = "NOT STARTED";
         }
     }
+    
+    private static void SetCategorySectionRecommendationDtoTag(CategorySectionRecommendationDto categorySectionRecommendationDto)
+    {
+        var recommendationReady = categorySectionRecommendationDto.RecommendationSlug != null;
 
-    private IEnumerable<CategorySectionDto> GetCategorySectionViewComponentViewModel(Category category)
+        categorySectionRecommendationDto.TagColour = recommendationReady ? TagColour.Blue : TagColour.Grey;
+        categorySectionRecommendationDto.TagText = recommendationReady ? "Ready" : "Not Available";
+    }
+
+    private async IAsyncEnumerable<CategorySectionDto> GetCategorySectionDto(Category category)
     {
         foreach (var section in category.Sections)
         {
-            var categorySectionDto = new CategorySectionDto()
+            var sectionStatus = category.SectionStatuses.FirstOrDefault(sectionStatus =>
```

Comment:
What happens after this if the section has never been started?

Reply:
Wait ignore me I see

Reply:
IMO might be worth creating another function for 99->102 that ... well does 99-102.

Then maybe do the refactoring around this component as we vaguely discussed before if you get time because it's a fair bit now.

YMMV

Reply:
Good shout, have made another function